fitbenchmarking/fitbenchmarking
# Update outdated main README

#### Description of Work

Fixes #422 

#### Testing Instructions

1. check that links on README work
2. and check that new text read ok


Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?
